mars/tests/test_utils.py

```diff
@@ -651,3 +651,58 @@ async def f2(x):
     with pytest.raises(CustomException):
         await utils.retry_callable(f2)(1)
     assert await utils.retry_callable(f2, ex_type=CustomException)(1) == 1
+
+
+def test_log_exception_wrapper():
+    log_file_name = "test_log_exception_wrapper.log"
+
+    try:
+        p = multiprocessing.Process(target=_log_exception, args=(log_file_name,))
+        p.start()
+        while p.is_alive():
+            time.sleep(0.1)
+        with open(log_file_name, "r") as f:
+            ex_log = f.read()
+            assert ex_log.count("raise_ex1") == 3, ex_log
+            assert ex_log.count("exception func1") == 1, ex_log
+            assert ex_log.count("raise_ex2") == 3, ex_log
+            assert ex_log.count("exception func2") == 1, ex_log
+    finally:
+        if os.path.exists(log_file_name):
+            os.remove(log_file_name)
+
+
+def _log_exception(log_file_name):
```

Comment:
It seems that the `log_exception_wrapper` code is not used. Can you remove it and the related tests from this PR? Then you can add them back in some PR when they are needed.

Reply:
reverted